File: src/Domain/Entities/Ganadero.cs
using TrazabilidadIberica.Domain.Common;

namespace TrazabilidadIberica.Domain.Entities;

public class Ganadero : BaseEntity
{
    public required string NombreRazonSocial { get; set; }
    public required string NIF { get; set; }
    public required string REGA { get; set; }
    public string? Telefono { get; set; }
    public string? Email { get; set; }
    public string? DireccionCompleta { get; set; }
    public string? IdentityUserId { get; set; }

    public ICollection<Finca> Fincas { get; set; } = new List<Finca>();
}

File: frontend/src/app/core/models/ganadero.ts
export interface Ganadero {
  id: string;
  nombreRazonSocial: string;
  nif: string;
  rega: string;
  telefono: string | null;
  email: string | null;
  direccionCompleta: string | null;
  identityUserId: string | null;
  createdOffline: boolean;
  clientId: string | null;
  createdAt: string;
  updatedAt: string;
  deletedAt: string | null;
}
